test/stats/train_mock.csv

```diff
@@ -0,0 +1,25 @@
+,filename,cyclo,cyclo_method_avg,cyclo_method_min,cyclo_method_max,npath_method_avg,npath_method_min,npath_method_max,ncss,ncss_method_avg,ncss_method_min,ncss_method_max,P1,P2,P3,P4,P5,P6,P7,P8,P9,P10,P11,P12,P13,P14,P15,P16,P17,P18,P19,P20_5,P20_7,P20_11,P21,P22,P23,P24,P25,P26,P28,P29,P30,P31,P32,M1,M2,M3_1,M3_2,M3_3,M3_4,M4,M5,M6,M7,lines_P1,lines_P2,lines_P3,lines_P4,lines_P5,lines_P6,lines_P7,lines_P8,lines_P9,lines_P10,lines_P11,lines_P12,lines_P13,lines_P14,lines_P15,lines_P16,lines_P17,lines_P18,lines_P19,lines_P20_5,lines_P20_7,lines_P20_11,lines_P21,lines_P22,lines_P23,lines_P24,lines_P25,lines_P26,lines_P28,lines_P29,lines_P30,lines_P31,lines_P32,halstead volume
```

Comment:
@lyriccoder Do we still need this file?

Reply:
Yes, we need it. We need to define test set for aibolit_stast. We can also make a mock inside code. but it will be very large.